demo/api/src/products/generated/product-variant.resolver.ts

```diff
@@ -1,14 +1,12 @@
 // This file has been generated by comet api-generator.
 // You may choose to use this file as scaffold by moving this file out of generated folder and removing this comment.
 
-import { RequiredPermission } from "@comet/cms-api";
 import { Parent, ResolveField, Resolver } from "@nestjs/graphql";
 
 import { Product } from "../entities/product.entity";
 import { ProductVariant } from "../entities/product-variant.entity";
 
 @Resolver(() => ProductVariant)
-@RequiredPermission(["products"], { skipScopeCheck: true })
```

Comment:
I suppose this shouldn't be removed, right?

Reply:
Basically it is a default when a query/mutation doesn't have it's own decorator. The default should not be all possible permissions. Maybe only read permission? Or no default at all?

Reply:
now it uses the same decorator as the single-query. as you will need nested properties especially when reading for some detail-page.

but I think it would also make sense to consider the CrudField({... input: true, ...}) or replace this with some Crud[Field|Nested]({...requiredPermission: ??}) decorator or specialized config...

Reply:
ok, then a `@CrudField({requiredPermission: ...})` would be appropriate (but doesn't have to be part of this PR imho)

Reply:
where do we collect and track suchlike follow-tasks? (and the other https://github.com/vivid-planet/comet/pull/1653#discussion_r1475809660) I'm sure I wont remember this once I close this tab.

would an issue with a correct label be appropriate?